samples/bencher/src/main.rs

```diff
@@ -0,0 +1,50 @@
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+use std::error::Error;
+use std::time::Instant;
+use wasmcloud_actor_core::serialize;
+use wasmcloud_actor_http_server as http;
+use wasmcloud_host::{Actor, HostBuilder};
+
+#[actix_rt::main]
+async fn main() -> Result<(), Box<dyn Error + Sync + Send>> {
+    let h = HostBuilder::new().build();
+    h.start().await?;
+    let echo = Actor::from_file("../../target/wasm32-unknown-unknown/release/bench_actor_s.wasm")?;
```

Comment:
Since this depends on running `make release` in the bench-actor directory, I think this could be better off as a match statement so we can provide some context to the failure in case someone just runs this sample. It will error with a "not found", but then the user will need to check out the code to figure out why. I think, in case of error, a message like:
```
"Error loading bench actor, please run `make release` in samples/bench-actor/"
```
will add some clarity.